tikv-configuration-file.md

```diff
@@ -1423,10 +1539,131 @@ Configuration items related to `raftdb`
 + Default value: `2`
 + Minimum value: `1`
 
+### `max-open-files`
+
++ The total number of files that RocksDB can open
++ Default value: `40960`
++ Minimum value: `-1`
+
+### `max-manifest-file-size`
+
++ The maximum size of a RocksDB Manifest file
++ Default value: `"20MB"`
++ Minimum value: `0`
++ Unit: B|KB|MB|GB
+
+### `create-if-missing`
+
++ Determines whether to automatically create a DB switch
++ Default value: `true`
+
+### `enable-statistics`
+
++ Determines whether to enable the statistics of RocksDB
```

Comment:
This config is under [raftdb] not [rockdb].

Reply:
Updated to `Raft RocksDB`.